clients/web/apps/marketing/src/styles/global.css

```
@@ -1,27 +1,20 @@
-@import url("@corvus/shared/theme.css");
+/* ─────────────────────────────────────────────────────────────
+   Corvus Marketing — Nothing Design System
+   Dark-first, monochrome, typography-driven.
+   ───────────────────────────────────────────────────────────── */
+
+/* ── Tokens + Base ── */
+@import url("@corvus/shared/nothing-theme.css");
```

Stylelint (17.6.0)

[error] Expected "url("@corvus/shared/nothing-theme.css")" to be ""@corvus/shared/nothing-theme.css"" (import-notation)

(import-notation)